plugins/in_tcp/tcp_config.c

```diff
@@ -126,6 +127,9 @@ int tcp_config_destroy(struct flb_in_tcp_config *ctx)
 {
     flb_sds_destroy(ctx->separator);
     flb_free(ctx->tcp_port);
+    if (ctx->server_fd > 0) {
```

Comment:
Note: If it doesn't close fd, the second and subsequent tests will fail since fluent-bit will not bind the port.